### PR TITLE
sm-55 - create card component

### PR DESCRIPTION
**Pull request by svipulc**
# Pull Request: Card Component 

## Description

This pull request introduces a reusable Card component to our application. The Card component is designed to display content in a visually appealing and organized manner, with the ability to customize the header, content, and footer sections.

### Features

- **Flexible Structure**: The Card component allows for a header, content, and footer section, enabling users to organize their content effectively.

### Default card
DefaultCard

### Post Card
post card

### CODE REVIEW

File: src/components/Card/index.style.tsx
Comment (kashyap27102) [inline src/components/Card/index.style.tsx:1]: Can you just remove the not needed part like
```js
{
    variants: {
      variant: {
        //variant
      },
      size: {
        //size
      },
    },
    defaultVariants: {
      //default variant
    },
  }
```
  or is it have meaning ?

File: .storybook/main.ts
Comment (kashyap27102) [inline .storybook/main.ts:1]: The commit includes alterations that do not align with the PR

File: postcss.config.js
Comment (kashyap27102) [inline postcss.config.js:1]: The commit includes alterations that do not align with the PR

**Review by kashyap27102 — CHANGES_REQUESTED** [review]

File: .storybook/main.ts
Comment (svipulc) [reply .storybook/main.ts:1]: this changes are done by eslint

File: postcss.config.js
Comment (svipulc) [reply postcss.config.js:1]: this changes are done by eslint

**Review by kashyap27102 — APPROVED** [review]

**Pull request merged**